includes/utils.php

```diff
@@ -312,20 +312,16 @@ public static function media_upload( $file_, $use_media_library = true, $check_i
 	public static function get_mimetype( $file ) {
 		$mime_type = null;
 
-		if ( function_exists( 'finfo_open' ) ) {
-			if ( $finfo = finfo_open( FILEINFO_MIME ) ) {
-				$mime_type = explode( ';', finfo_file( $finfo, $file ) );
-				$mime_type = trim( $mime_type[0] );
-				finfo_close( $finfo );
-			}
+		if ( function_exists( 'mime_content_type' ) ) {
+            $mime_type = mime_content_type( $file );
 		}
 
 		if ( null === $mime_type ) {
 			$type_info = wp_check_filetype( $file, wp_get_mime_types() );
 			$mime_type = $type_info['type'];
 		}
 
-		return $mime_type;
+		return esc_attr( $mime_type );
```

Comment:
We should be sanitizing here instead of escaping since this in incoming data rather than echoing.

Reply:
Thanks, I have added sanitization :)